casebased/components/similarity_measure/similarity.py

```diff
@@ -28,14 +33,16 @@ def fit(self, **kwargs):
                 self._fit_classifier(**kwargs)
         self._fit_classifier(**kwargs)
 
-    def _fit_classifier(self, **kwargs):
-        k = kwargs.get("k")
+    def _fit_classifier(self, case_base: CaseBase, vocabulary: Vocabulary, **kwargs):
+        k = kwargs.get("k") if kwargs.get("k") else self.k
+        if k is None:
+            k = "auto"
```

Comment:
```suggestion
```
Kann gelöscht werden wenn k by default "auto" ist. Siehe anderes Kommentar in Review. Bei k hatte ich mir auch vorgestellt, dass das k für ein System gespeichert wird und nicht jedes mal neu berechnet werden muss. Das ist aber Scope von einem anderen Ticket.